### PR TITLE
Team & Mentors Section

### PR DESCRIPTION
**Pull request by ckvignesh**
Hi @evedes,

As you had elaborated in Issue #23, I have removed the team section for now (commented the code out so it can be used in future).

I'll work on the 2nd point (improving members cards) as discussed before :)

Please let me know if you require any changes to the code.

Thanks.

### CODE REVIEW

**Comment by evedes** [comment]
Hey @ckvignesh, good job! Please remove all the unused code. We'll rewrite it better and soon! ❤️

**Comment by ckvignesh** [comment]
> Hey @ckvignesh, good job! Please remove all the unused code. We'll rewrite it better and soon! heart

Hi @evedes. Thanks for reviewing. I've removed the unused code so we can write a better one, later. Kindly review the commit...

**Review by ckvignesh — COMMENTED** [review]
Removed unused code for team...

File: pages/index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import Head from 'next/head'
-import { About, Challenges, Mentors, Team, Tldr } from '../sections'
+import { About, Challenges, Mentors, Tldr } from '../sections' // add Team here while adding it later. also add the team files to Sections folder.
```
Comment (evedes) [inline pages/index.js:2]: Remove comments, please.
We should only comment whys, not hows. That's a good rule of thumb.

File: pages/index.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ export default function Home() {
       <About />
       <Challenges />
       <Mentors />
-      <Team />
+      {/* add team info here */}
```
Comment (evedes) [inline pages/index.js:22]: Remove comment please!

File: sections/index.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import About from './About'
 import Challenges from './Challenges'
 import Mentors from './Mentors'
-import Team from './Team'
+// import Team from './Team'
```
Comment (evedes) [inline sections/index.js:4]: Remove comment please!

File: sections/index.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
 import Tldr from './Tldr'
 
-export { About, Challenges, Mentors, Team, Tldr }
+export { About, Challenges, Mentors, Tldr } //add team here when it's ready
```
Comment (evedes) [inline sections/index.js:7]: Remove comment please!

**Review by evedes — COMMENTED** [review]
Hey, good job with this! ❤️ 

As a rule of thumb, no need to comment hows, only whys.
This means developers don't need to read a comment that tells them, add this here or there. That's their criteria, programming is a craft.

So, just requesting you to remove the comments and we're good to merge!

**Comment by ckvignesh** [comment]
> Hey, good job with this! heart
> 
> As a rule of thumb, no need to comment hows, only whys. This means developers don't need to read a comment that tells them, add this here or there. That's their criteria, programming is a craft.
> 
> So, just requesting you to remove the comments and we're good to merge!

Thanks, @evedes. I've removed the comments.

**Comment by ckvignesh** [comment]
Thanks for merging, @evedes.

**Pull request merged**